.pipelines/templates/build-python.yml

```diff
@@ -0,0 +1,32 @@
+parameters:
+    - name: SERVICE
+    - name: WORKING_FOLDER
+
+steps:
+    - task: Bash@3
+      displayName: 'Linting: ${{ parameters.SERVICE }}'
+      inputs:
+          targetType: 'inline'
+          workingDirectory: ${{ parameters.WORKING_FOLDER }}
+          script: |
+              pipenv run flake8 ${{ parameters.WORKING_FOLDER }}
+
+    - task: Bash@3
+      displayName: 'Unit tests: Analyzer'
+      inputs:
+          targetType: 'inline'
+          workingDirectory: ${{ parameters.WORKING_FOLDER }}
+          script: |
+              # Install pytest and run tests
+              pipenv install --dev --skip-lock pytest-azurepipelines

```

Comment:
is that used for publishing results to azure pipelines in a better manner?

Reply:
there's a publish artifact task, and we can use pipeline feed. I did not get into it. debating what would be the best option. Also for the wheel

Reply:
yes I saw that it was merely a copy, took another look and I think it's worth the installation

Reply:
lets address this in a seperate PR.